### DISCUSSION TITLE
Thảo luận mới

### DISCUSSION THREAD

**Discussion by LeHieu199644** [community/community#103022] [Code Security]
### Select Topic Area

General

### Body

Mới dùng nên ko biết

**Comment by ghostinhershell** [comment]
Welcome to GitHub and the GitHub Community @LeHieu199644 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

I do want to note that, unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

Best of luck!
